ftw/simplelayout/tests/builders.py

```diff
@@ -11,6 +13,12 @@ class ContenPageBuilder(DexterityBuilder):
 class TextBlockBuilder(DexterityBuilder):
     portal_type = 'ftw.simplelayout.TextBlock'
 
+    def with_dummy_image(self):
+        image = Path(__file__).joinpath('..', 'assets', 'fullhd.jpg').abspath()
```

Comment:
Is it relevant that the image is this big? A smaller image might speed up the tests a little :wink: 
How about `mario.gif`?